less/index.less

```diff
@@ -1 +1,33 @@
-// Follow the order in the readme
+@import "variables.less";
+@import "navigation.less";
+@import "header.less";
+@import "section.less";
+@import "footer.less";
+
+hr {
+    border-top: dashed 1.5px lightgray;
+}
+
+h1, h2, h3, h4 {
+    font-family: 'Indie Flower';
+    font-weight: normal;
+    color: black;
+}
+
+a {
+    color: inherit;
+    text-decoration: none;
+}
+
+@media (min-width: @desktop-min-width) {
+    .container {
+        width: 50%;
+        margin: 0 auto;
+    }
+}
+
+@media (max-width: @mobile-width) {
+    .container {
+        margin: 0px 10px;
+    }
+}
```

Comment:
index.less should just have the imports in the correct order. Styling should go in your other less files.